src/sentry/reprocessing.py

```diff
@@ -1,15 +1,11 @@
 import logging
-import uuid
 from collections.abc import Mapping
 from typing import Any
 
-import sentry_sdk
-
-from sentry.models.options.project_option import ProjectOption
-
 # Is reprocessing on or off by default?
 REPROCESSING_DEFAULT = False
 REPROCESSING_OPTION = "sentry:reprocessing_active"
+# TODO: make sure to clean these option up at some point:
```

Comment:
Can we be a little bit more clear about this comment?

Reply:
since this is a project option, we should bulk-remove these from the DB at some point.
I created an issue for that: https://github.com/getsentry/team-processing/issues/160